test/unit/bulk/common.test.js

```diff
@@ -1,132 +0,0 @@
-/* eslint-disable no-loss-of-precision */
-/* eslint-disable @typescript-eslint/no-loss-of-precision */
-// TODO(NODE-3774): Lower the integer literals below JS max precision
```

Comment:
Can we safely close the linked TODO?  I think so 🙂

Reply:
Agree - closed that ticket now.